.commitlintrc.json

```diff
@@ -0,0 +1,126 @@
+{
```

Comment:
Hey @cpcloud I was speaking to @jacques-n about the need to add the commit lint to this PR. My understanding is that its required to help semver decide if its a major or minor release. Is that correct?

Would you mind just saying a few words so that future us knows why we did this?

Reply:
> Hey @cpcloud I was speaking to @jacques-n about the need to add the commit lint to this PR. My understanding is that its required to help semver decide if its a major or minor release. Is that correct?

`commitlint` per se isn't necessary for release, but it will prevent commits that don't conform to semantic-release and therefore couldn't be used to compute a new release version.